reflex/components/core/layout/stack.py

```diff
@@ -0,0 +1,64 @@
+"""Stack components."""
+from __future__ import annotations
+
+from typing import Literal, Optional
+
+from reflex.components.component import Component
+from reflex.components.el.elements.typography import Div
+
+LiteralJustify = Literal["start", "center", "end"]
+LiteralAlign = Literal["start", "center", "end", "stretch"]
+
+
+class Stack(Div):
+    """A stack component."""
+
+    @classmethod
+    def create(
+        cls,
+        *children,
+        justify: Optional[LiteralJustify] = "start",
+        align: Optional[LiteralAlign] = "center",
+        spacing: Optional[str] = "0.5rem",
+        **props,
+    ) -> Component:
+        """Create a new instance of the component.
+
+        Args:
+            *children: The children of the stack.
+            justify: The justify of the stack elements.
+            align: The alignment of the stack elements.
+            spacing: The spacing between each stack item.
+            **props: The properties of the stack.
+
+        Returns:
+            The stack component.
+        """
+        style = props.setdefault("style", {})
+        style.update(
+            {
+                "display": "flex",
+                "alignItems": align,
+                "justifyContent": justify,
+                "gap": spacing,
+            }
+        )
```

Comment:
```suggestion
        props["style"] = {
                "display": "flex",
                "alignItems": align,
                "justifyContent": justify,
                "gap": spacing,
                **style,
        }
```

we should allow the user to override the styles right?